### ISSUE TITLE
Rework the way we build artifacts

### ISSUE THREAD

**Issue by Itxaka** [rancher/elemental#169]
Currently we have a mess in the way we build the artifacts for the repo. Some mix of docker in docker, dapper, strange targets, packer inside docker inside dapper, config files embedded in bash scripts and so on.

This makes it very difficult to understand  the pipelines and understand where the artifacts are coming from, what are we building and what is the final product.

We should be able to drop the ros-image-build and all the dapper files and recrete decent targets to build from. I.E. packer templates to build the qcow+box, a Dockerfile to build the container and the iso, which is already in the Makefile.

Then we could create a clean CI pipeline in which every built thing its own job so we can clearly see what is doing and understan when there is failures.

Also, lets stop pushing to ssh.tl for CI as those images are dropped after a few hours. We can create an elemental-ci repo and push the images there as we do with the operator so we can test them and come back.

Also a decent release strategy should be created.

Basically, lets see what we have already on the operator, toolkit, cli and use it in this repo to bring it up to standars with the rest of our projects instead of having a black box that nobody knows about.

**Comment by ldevulder** [comment]
In cas it could help a bit :-), here what I found: https://github.com/rancher/elemental/pull/93#issuecomment-1156292431

**Comment by Itxaka** [comment]
gonna claim this as done, there is card for further steps
